MediaBrowser.Providers/Plugins/MusicBrainz/AlbumProvider.cs

```diff
@@ -74,6 +74,12 @@ public class MusicBrainzAlbumProvider : IRemoteMetadataProvider<MusicAlbum, Albu
         /// <inheritdoc />
         public async Task<IEnumerable<RemoteSearchResult>> GetSearchResults(AlbumInfo searchInfo, CancellationToken cancellationToken)
         {
+            // TODO maybe remove when artist metadata can be disabled
+            if (!Plugin.Instance.Configuration.Enable)
```

Comment:
These are an ugly hack imo

Reply:
I agree but there's no easy alternative at the moment to disable this provider.